x-pack/plugins/security_solution/server/endpoint/lib/artifacts/task.ts

```diff
@@ -87,7 +91,7 @@ export class ManifestTask {
         params: { version: ManifestTaskConstants.VERSION },
       });
     } catch (e) {
-      this.logger.debug(`Error scheduling task, received ${e.message}`);
+      this.logger.error(new EndpointError(`Error scheduling task, received ${e.message}`, e));
```

Comment:
are these logs you moved from `debug` going to be noisy?

Reply:
These I think should be reported and not require a stack to turn on verbose messaging. We depend on this task running in order for the policies/Endpoints to remain updated with latest set of user generated artifacts. I felt that if we can't schedule the task to run, then that a significant error